test/fastifyCsrf.test-d.ts

```diff
@@ -0,0 +1,26 @@
+import * as fastify from "fastify";
+import * as fastifyCookie from "fastify-cookie";
+import * as fastifyCSRF from "../lib/fastifyCsrf";
+
+const server = fastify();
+server.register(fastifyCookie);
+server.register(fastifyCSRF, {
+  cookie: true,
+  ignoreMethods: ["GET"],
+  key: "_custom_csrf",
+  value: (req) => {
+    return req.body["some_key"];
+  },
+});
+
+server.get("/", (request, reply) => {
+  var form = `
+      <form method = "post" action="/data">
+         <input type="text" name"field_name"/>
+         <input type="hidden" value="${request.csrfToken()}"
+	 name="_csrf /* this token can be sent in request header as well */"/>
```

Comment:
I can see you have copy-pasted this from README :).

Unfortunately, JS comments (`/* */`) do not play nicely with HTML, could you, please, remove it?

Reply:
Yeah good, point 👍 See 188442c